### PR TITLE
add rhumb_bearing

### PR DESCRIPTION
**Pull request by SteffenHaeussler**
add rhumb_bearing and finalize https://github.com/diogomatoschaves/pyturf/issues/9

Please make sure that:

- [x] You have read [CONTRIBUTING.md](../CONTRIBUTING.md)
- [x] You Include a link to the issue this pull request is associated to.
- [x] The linter has been run at the root level of the project: `black .`
- [x] All tests are passing: `python -m pytest --verbose --cov=./` 
- [x] Your branch is up to date with master: `git rebase upstream/master`

For new modules, also make sure that:

- [x] The new module function is exported in `turf/<your-module>/__init__.py` and `turf/__init__.py`.
- [x] The new module has been added under the `Available Modules` section on the [README](../README.md).

### CODE REVIEW

File: turf/rhumb_bearing/_rhumb_bearing.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+import numpy as np
+
+from turf.helpers import degrees_to_radians, radians_to_degrees
+from turf.helpers import Point
+from turf.invariant import get_coord
+
+
+def rhumb_bearing(
+    origin: Union[List, Point], destination: Union[List, Point], options: Dict = None
```
Comment (diogomatoschaves) [inline turf/rhumb_bearing/_rhumb_bearing.py:12]: The types here should be `Union[Sequence, Dict, Feature]`

File: turf/rhumb_bearing/_rhumb_bearing.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    Takes two {Point|points} and finds the bearing angle between them along a
+    Rhumb line
+     * i.e. the angle measured in degrees start the north line (0 degrees)
+
+    # https://en.wikipedia.org/wiki/Rhumb_line
+
+    :param start: starting point [lng, lat] or Point feature
+    :param end: ending point [lng, lat] or Point feature
+    :param options: Optional parameters
+        :param final: calculates the final bearing if True
```
Comment (diogomatoschaves) [inline turf/rhumb_bearing/_rhumb_bearing.py:24]: here should maybe be `options["final"]: Calculates the final bearing if True`

File: turf/rhumb_bearing/_rhumb_bearing.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    if final:
+        bearing = calculate_rhumb_bearing(destination, origin)
+    else:
+        bearing = calculate_rhumb_bearing(origin, destination)
+
+    return bearing
+
+
+def calculate_rhumb_bearing(origin: List, destination: List) -> float:
```
Comment (diogomatoschaves) [inline turf/rhumb_bearing/_rhumb_bearing.py:43]: change to (`origin: Sequence, destination: Sequence`)

File: turf/rhumb_bearing/_rhumb_bearing.py (hunk 1 above)
Comment (SteffenHaeussler) [reply turf/rhumb_bearing/_rhumb_bearing.py]: 👍

File: turf/rhumb_bearing/_rhumb_bearing.py (hunk 2 above)
Comment (SteffenHaeussler) [reply turf/rhumb_bearing/_rhumb_bearing.py]: 👍

File: turf/rhumb_bearing/_rhumb_bearing.py (hunk 3 above)
Comment (SteffenHaeussler) [reply turf/rhumb_bearing/_rhumb_bearing.py]: 👍

**Pull request merged**